cmd/event-notifier.go

```diff
@@ -291,7 +292,7 @@ func eventNotifyForBucketNotifications(eventType, objectName, bucketName string,
 		eventMatch := eventMatch(eventType, qConfig.Events)
 		ruleMatch := filterRuleMatch(objectName, qConfig.Filter.Key.FilterRules)
 		if eventMatch && ruleMatch {
-			targetLog := globalEventNotifier.GetExternalTarget(qConfig.QueueARN)
+			targetLog := globalEventNotifier.GetExternalTarget(unmarshalSqsARN(qConfig.QueueARN).String())
```

Comment:
@harshavardhana, I think it is better to update qConfig.QueueARN just after ARN validation, I think it would be cleaner that way.